src/Table/index.jsx

```diff
@@ -1,16 +1,110 @@
 import React from 'react';
 import classNames from 'classnames';
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+import isRequiredIf from 'react-proptype-conditional-require';
 import PropTypes from 'prop-types';
 
 import styles from './Table.scss';
+import Button from '../Button';
 
 class Table extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = props.tableSortable ?
+      {
+        sortedColumn: this.props.defaultSortedColumn,
+        sortDirection: this.props.defaultSortDirection,
+      }
+      :
+      {
+        sortedColumn: '',
+        sortDirection: '',
+      };
```

Comment:
This looks a lot nice. I think I'll do the ternary, just because that's what I've used before, but if it's stylistically more consistent for me to use the short-circuit, I can do that as well. 

I think I just didn't realize I could do it for each individual attribute of the state and had to do on the actual object level, which is why I had the ugly way of doing it.

Reply:
Short-circuits are janky and some purists (not me tho) frown on them, so the ternary is a good instinct 😛